Kitodo-API/src/main/java/org/kitodo/api/dataformat/service/AreaServiceInterface.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat.service;
+
+/**
+ * Interface for a service that handles access to the {@code <mets:area>}
+ * element.
+ *
+ * <p>
+ * An area is a view on a media unit (METS idiom: file). For a still image, this
+ * is often a rectangle which is left over when cutting off the top, bottom,
+ * right, and left sides of the image. For other media formats, however, this
+ * can also be other properties, such as the time for the start and end of the
+ * playback for a sound or video sequence.
+ *
+ * <p>
+ * This part is provided here only insofar as there is one area per page that
+ * currently has no attributes apart from a reference to the file. This is
+ * intended to be extended here later for the support of Area. In the future,
+ * there may be several areas per page, and the areas will have other attributes
+ * that describe their positioning.
+ */
+public interface AreaServiceInterface {
+    /**
+     * Returns a service to access the media unit ({@code <mets:file>} element)
+     * on which the view is based.
+     *
+     * @return a service to access the file
+     */
+    FileServiceInterface getFile();
```

Comment:
It is confusing - File or FileService? Method name says one, parameter says second.

Reply:
You will see that all methods return services (and of course strings, etc.), and in the names of the methods the "service" does not appear anywhere. This is because the interface should be named according to METS so that, knowing METS, it is intuitively understood. If I could decide that for myself, I would call the interfaces and methods quite differently, not even "File", because it's not a file at all. If you are not at home in METS, this seems strange at first glance, but it is primarily aimed at people who deal with METS on a daily basis. I have tried to describe in the javadocs what the objects mean exactly. You'll see that it's something completely different from what you would expect if you did not think in METS. But I've decided to name the classes that implement the interface differently, to better understand what they mean. I hope that's enough for both sides.